typescript/infra/config/environments/testnet4/aw-validators/hyperlane.json

```diff
@@ -7,10 +7,14 @@
     ]
   },
   "arbitrumsepolia": {
-    "validators": ["0x09fabfbca0b8bf042e2a1161ee5010d147b0f603"]
+    "validators": [
```

Comment:
sadly the serialization of these fights prettier, think you'll need to run `yarn prettier`